src/Routes/ArticleComments.jsx

```diff
@@ -0,0 +1,33 @@
+import { getArticleComments } from "../../utils";
+import { Link, Route, Routes } from "react-router-dom";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+const ArticleComments = () => {
+  const [comments, setComments] = useState([]);
+  const id = useParams();
+  useEffect(() => {
+    getArticleComments(id.article_id).then((res) => {
+      setComments(res.data.comments);
+    });
+  }, []);
+  return (
+    <>
```

Comment:
While comments load we could offer some feedback to the users, a bit like you have done for the whole page 🙂